src/test/regress/citus_tests/common.py

```diff
@@ -94,6 +94,7 @@ def get_pg_major_version():
     15: "11.1.5",
     16: "12.1.5",
     17: "13.0.1",
+    18: "13.1.1",
```

Comment:
13.1.0 here since we didn't have a patch release on 13.1 yet.

Reply:
updated